### PR TITLE
Officially linking to new sig-community

### PR DESCRIPTION
**Pull request by quaid**
- now in table of contents
- should appear in auto-generated docs

Signed-off-by: Karsten Wade <kwade@redhat.com>

### CODE REVIEW

File: sigs.yaml [hunk 1]
```diff
@@ -42,35 +45,15 @@ sigs:
...
-    frequency: Bi-Weekly
-    url: meet.google.com/zfj-woub-xwp
+    frequency: weekly
+    url: tbd
   contact:
-    slack: sig-operations
+    slack: operations
+    mailing_list: community@lists.operate-first.cloud
   subprojects:
-  - name: cluster-ops
```
Comment (larsks) [inline sigs.yaml:50]: Why does this PR remove all the sig-operations subproject data? This change isn't mentioned in either the PR description or the comments.

**Review by larsks — CHANGES_REQUESTED** [review]

File: sigs.yaml (hunk 1 above)
Comment (quaid) [reply sigs.yaml:50]: This change and the next one are not intentional. It appears that despite fetching and rebasing on upstream, I managed to have a sigs.yaml file that was not entirely what is in upstream

Upstream: https://github.com/operate-first/community/commits/main/sigs.yaml
Mine: https://github.com/quaid/community/commits/quaid-add-sig-community-to-toc/sigs.yaml

I'm not sure where it got trimmed out, if I could find that point I suppose I can rebase there and that would cascade down to this PR?

Alternately I can just add it in and move on. :)

File: sigs.yaml [hunk 2]
```diff
@@ -42,35 +45,15 @@ sigs:
...
-      day: Thursday
-      time: "8:30"
-      tz: EST
-      frequency: Weekly
-      url: meet.google.com/wxh-pizv-vwt
-  - name: ops-docs
-    description: |-
-      Manage documentation for the Operate First Cloud offering. Ensure documentation is created
-      and maintained in coordination with other Operate First sigs.
+  - name: tbd
```
Comment (quaid) [inline sigs.yaml:54]:
```suggestion
  - name: cluster-ops
```

File: sigs.yaml [hunk 3]
```diff
@@ -45,32 +48,12 @@ sigs:
     frequency: Bi-Weekly
     url: meet.google.com/zfj-woub-xwp
   contact:
-    slack: sig-operations
+    slack: operations
```
Comment (quaid) [inline sigs.yaml:51]:
```suggestion
    slack: sig-operations
```

File: sigs.yaml [hunk 4]
```diff
@@ -46,31 +49,11 @@ sigs:
     url: meet.google.com/zfj-woub-xwp
   contact:
     slack: sig-operations
+    mailing_list: community@lists.operate-first.cloud
   subprojects:
   - name: cluster-ops
```
Comment (quaid) [inline sigs.yaml:54]:
```suggestion
  - name: cluster-ops
      description: Day to day operations of all the clusters that are part of the Operate First Cloud Offering.
    owners:
    - https://raw.githubusercontent.com/operate-first/community/main/sig-operations/subprojects/cluster-ops/OWNERS
    meetings:
    - description: Cluster Ops meeting
      day: Tuesday
      time: "8:30"
      tz: EST
      frequency: Weekly
      url: meet.google.com/wxh-pizv-vwt
    - description: Cluster Ops meeting
      day: Thursday
      time: "8:30"
      tz: EST
      frequency: Weekly
      url: meet.google.com/wxh-pizv-vwt
  - name: ops-docs
    description: |-
      Manage documentation for the Operate First Cloud offering. Ensure documentation is created
      and maintained in coordination with other Operate First sigs.
```

File: sigs.yaml [hunk 5]
```diff
@@ -46,31 +49,11 @@ sigs:
...
-      tz: EST
-      frequency: Weekly
-      url: meet.google.com/wxh-pizv-vwt
-  - name: ops-docs
-    description: |-
-      Manage documentation for the Operate First Cloud offering. Ensure documentation is created
-      and maintained in coordination with other Operate First sigs.
     owners:
-    - https://raw.githubusercontent.com/operate-first/community/main/sig-operations/subprojects/ops-docs/OWNERS
+    - https://raw.githubusercontent.com/tbc/tbc/main/tbc/OWNERS
```
Comment (quaid) [inline sigs.yaml:76]:
```suggestion
    - https://raw.githubusercontent.com/operate-first/community/main/sig-operations/subprojects/ops-docs/OWNERS
```

File: sigs.yaml (hunk 1 above)
Comment (quaid) [reply sigs.yaml:50]: OK, it's the same except I removed a line break that appeared in the middle of a sentence. It appears I also have a mailing list identified, are we not using that field? I can remove it too.

File: sigs.yaml [hunk 6]
```diff
@@ -46,10 +49,10 @@ sigs:
     url: meet.google.com/zfj-woub-xwp
   contact:
     slack: sig-operations
+    mailing_list: community@lists.operate-first.cloud
   subprojects:
   - name: cluster-ops
-    description: Day to day operations of all the clusters that are part of the Operate
-      First Cloud Offering.
+      description: Day to day operations of all the clusters that are part of the Operate First Cloud Offering.
```
Comment (durandom) [inline sigs.yaml:55]: please dont touch this line, now it's indented wrong

**Review by durandom — CHANGES_REQUESTED** [review]

File: sigs.yaml (hunk 6 above)
Comment (quaid) [reply sigs.yaml]: Fixed, thanks for the catch and the etiquette lesson.

**Review by durandom — APPROVED** [review]
/approve
/lgtm

**Pull request merged**